src/main/java/com/example/todo_restapi/service/TaskService.java

```diff
@@ -39,11 +44,11 @@ public List<TaskDto> getAll() {
         return taskDto;
     }
 
-    public Optional<TaskDto> updateTask(Long id, TaskDto taskDto) {
+    public Optional<TaskDto> endTask(Long id) {
```

Comment:
Nie rozumiem czemuendTask ma coś zwracać. Powinno moim zdaniem być void lub zwróc zakończone zadanie.

Jeżeli brak zadania to rzuc błędem